src/sentry/tasks/post_process.py

```diff
@@ -16,6 +16,8 @@
 from sentry.issues.grouptype import GroupCategory
 from sentry.issues.issue_occurrence import IssueOccurrence
 from sentry.killswitches import killswitch_matches_context
+from sentry.sentry_metrics.kafka import KafkaMetricsBackend
```

Comment:
@ayirr7 I think we should hide which backend gets used from the caller.

it should be something like

```python
from sentry.sentry_metrics import get_sentry_metrics_backend

# today this returns a KafkaMetricsBackend, but later it
# should be configurable from somewhere outside this file
metrics_backend = get_sentry_metrics_backend() 
```

This also makes it easier to only have one connection (pool) per process

Reply:
Sentry has the service delegator / LazyServiceWrapper abstraction which is commonly used for this